ui.tests/test-module/specs/recaptcha/recaptcha.authoring.spec.js

```diff
@@ -63,6 +63,30 @@ describe('Page - Authoring', function () {
              .should("exist");
 
         // Checking some dynamic behaviours
+        cy.get("coral-select[name='./rcCloudServicePath']").click().then(() => {
```

Comment:
Please don't make use of coral selectors in the test case, can we make sure that the selectors are specific to our use-case via BEM

Reply:
Resolved by using openPage API

Reply:
This selector is now replaced by the bem class. However, the selector to switch the config is still a coral selector.
When a coral select is clicked, the items come as a popover and do not have the parent class. I am seeing if I can resolve that too, in the meantime. Do let me know if you have any suggestions for the same